bf2fc6cc711aee1a0c2a/must-gather
# add initial must-gather script and dockerfile

Add initial must-gather script. This script will dump:

- All base k8s resources for fleetshard, strimzi, observability, ingress and any managedkafka namespace
- All observability CRs in the managed-application-services-observability namespace
- All managedkafka and kafka crs
- Ingresscontroller crs
- Additional scrape config secret

To Test:

1. Ping me for cluster details
4. OC login to cluster
5. Run `oc adm must-gather --image=quay.io/stobin/mk-must-gather:0.0.9`
6. Check output of must-gather.local.< UID >
